lib/node_modules/@stdlib/string/base/truncate-middle/README.md

```diff
@@ -0,0 +1,96 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2023 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# truncateMiddle
+
+> Truncate UTF-16 code units of string in the middle to a specified length.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var truncateMiddle = require( '@stdlib/string/base/truncate-middle' );
+```
+
+#### truncateMiddle( str, len\[, seq] )
```

Comment:
For "base" packages, we tend to favor non-variadic interfaces. As such, I think we should make `seq` mandatory here. Higher level consumers can explicitly provide a default `seq`.

Reply:
Once made mandatory, all examples, tests, and benchmarks should be updated accordingly.